python/xir/program.py

```diff
@@ -1,12 +1,14 @@
+"""This module contains the XIRProgram class and classes for the Xanadu IR"""
+
 import re
 import warnings
 from decimal import Decimal
-from typing import Any, Dict, List, Sequence, Set, Tuple, Union
+from typing import Any, Collection, Dict, List, Mapping, Sequence, Tuple, Union
 
 from .decimal_complex import DecimalComplex
 from .utils import strip
 
-"""This module contains the XIRProgram class and classes for the Xanadu IR"""
+Wire = Union[int, str]
 
 Wire = Union[int, str]
```

Comment:
There's now an extra `Wire` type here. 😉

Reply:
...and this file isn't big enough for the both of them!